lib/purchases_flutter.dart

```diff
@@ -43,13 +43,17 @@ class Purchases {
   static final Set<ReadyForPromotedProductPurchaseListener>
       _readyForPromotedProductPurchaseListeners = {};
 
+  static CustomerInfo? _lastReceivedCustomerInfo;
```

Comment:
I don't like static variables like this, that's why I think this is not the preferred approach. But if we consider it a temporal solution and plan to solve it better in the future, I would be ok with it.